docs/src/guide/installation.md

```diff
@@ -34,10 +34,46 @@ Please help us [keep these instructions up-to-date][contribute]!
 
 For Ubuntu 20.04LTS, these commands install the basic build system:
 
-	sudo apt-get install python3 python3-pip scons git gcc-arm-none-eabi openocd \
-	  gcc build-essential libboost-all-dev
+	sudo apt install python3 python3-pip scons cmake git doxygen
 	pip3 install modm gdbgui
 
+!!! info "Python Packages in PATH"
+	The pip3 command installs executables into `~/.local/bin`, which
+	must be added to PATH if it is not already the case.
+	Add the following line to the end of your `~/.bashrc` file:
+
+		export PATH="~/.local/bin:$PATH"
+
+For the host system (x68_64/arm64) as a target:
+
+	sudo apt install gcc build-essential libboost-all-dev
+
+For ARM Cortex-M targets:
+
+	sudo apt install openocd
```

Comment:
Is this OpenOCD release up-to-date?

Reply:
Seeing the URLs below, should we just add the download and install instructions for OpenOCD too?

Reply:
> Is this OpenOCD release up-to-date?

Up-to-date for >99% of all modm targets, should work for most people.

Building OpenOCD from Source (or with additional patches) is to complicated for this guide. Maybe we can link to e.g. https://rleh.de/2019/10/08/openocd-stm32-stm32g4-stm32g0.html?

Reply:
Yeah, good point. Homebrew hides all the dependencies for source compilation, so I forgot about the ugly. Let's link to your blog instead!